plugins/modules/nxos_command.py

```diff
@@ -60,8 +60,9 @@
   retries:
     description:
     - Specifies the number of retries a command should by tried before it is considered
-      failed.  The command is run on the target device every retry and evaluated against
+      failed. The command is run on the target device every retry and evaluated against
       the I(wait_for) conditionals.
+    - The commands are run once when I(retries) is set to C(0).
     default: 10
```

Comment:
Making this change arguably retains the default behavior, despite the different value.
```suggestion
    default: 9
```